src/hooks/search/utils.ts

```diff
@@ -52,10 +69,14 @@ const searchItemToContentItem = ({
     };
 };
 
-export const getFilters = async (contentType?: PillCategory) => {
+export const getFilters = async (
+    contentType?: PillCategory,
+    allSearchContent?: SearchItem[]
+) => {
     const allFilters = contentType === undefined;
-    const allContent: SearchItem[] = await getAllSearchContent();
-
+    const allContent: SearchItem[] = allSearchContent
```

Comment:
`allSearchContent || await getAllSearchContent()`

Reply:
Good catch on this one